docs/REST API/rest-api.md

```diff
@@ -6269,7 +6343,7 @@ Returns a list of global queries or team queries.
 | order_direction | string  | query | **Requires `order_key`**. The direction of the order given the order key. Options include `asc` and `desc`. Default is `asc`. |
 | team_id         | integer | query | _Available in Fleet Premium_. The ID of the parent team for the queries to be listed. When omitted, returns global queries.                  |
 | query           | string  | query | Search query keywords. Searchable fields include `name`.                                                                      |
-
+| merge_inherited | boolean | query | _Available in Fleet Premium_. If `true`, will include global queries in addition to team queries when filtering by `team_id`. (If no `team_id` is provided, this parameter is ignored.)
```

Comment:
@jacobshandling was your question related to this param?  ie.  what should happen if no team_id is sent and `merge_inherited` is sent?

Reply:
@mostlikelee exactly

Reply:
> (If no `team_id` is provided, this parameter is ignored.)

Means it shouldn't error— since that wasn't obvious, if y'all have thoughts about how to make that behavior clearer in the description, please suggest a change!

Reply:
@rachaelshaw thanks for clarifying – it is pretty obvious actually.